docs/manual/creating-a-checker.tex

```diff
@@ -1783,6 +1783,10 @@
 already reported the bug, and you want to continue using the checker on a
 large codebase without being inundated in stack traces.
 
+\item \code{-AdumpOnErrors}: Outputs the stack trace of the program when the
+Checker Framework encounters an error or warning. The option also requires a
+boolean parameter to enables its functionality (e.g. \code{-AdumpOnErrors=true}).
```

Comment:
The boolean shouldn't be required, see the other boolean flags above. Just passing `-AdumOnErrors` should be equal to `-AdumOnErrors=true`. The user could turn off the flag using `-AdumOnErrors=false`.